config.properties.template

```diff
@@ -70,6 +70,10 @@ auto_fill_incubator=true
 # Should the bot EVER drop items (true/false)
 drop_items=true
 
+# Should the bot wait between each item/group dropped? Time in milliseconds, will be randomized a little, to skip use -1
```

Comment:
What would be a realistic (= ingame) value for this? Maybe put that value in this description so people can choose to use this value?